quality-assurance/user-acceptance-testing.md

```diff
@@ -0,0 +1,41 @@
+## What is User Acceptance Testing
+
+UAT means testing software against its business requirements i.e. does it solve the business problem as intended?
+
+## Who carries out UAT
+
+Business users are best placed to do UAT, as they are aware of the reality of their business, business processes, roles etc.  Sometimes, a tester will act in place of users.
```

Comment:
this needs to be structured and ideally captured so that its repeatable by others within the teams. it doesn't have to be Test Plans within Azure but this is an ideal way to track changes to the test scripts and an easy mechanism for reporting execution results